theme_clean/static/src/scss/primary_variables.scss

```diff
@@ -195,9 +195,6 @@ $o-theme-font-configs: (
 $o-website-values-palettes: (
     (
         'color-palettes-name': 'clean-1',
-        'headings-font': 'Dosis',
-        'navbar-font': 'Dosis',
-        'buttons-font': 'Dosis',
```

Comment:
This shouldn't have been removed ?

Reply:
Whoa, good catch, [I restored it](https://github.com/odoo/design-themes/compare/3687dea6e6c879300db90244b87bd52a055ba4c4..502d46cc9dd20c14c6e22b8bb4e36fa872c1c8cd) :+1: